docs/reference/method/MongoDBCollection-explain.txt

```diff
@@ -29,7 +29,7 @@ Definition
 Parameters
 ----------
 
-``$explainable`` : :phpclass:`MongoDB\Operation\Explainable`
+``$explainable`` : ``MongoDB\Operation\Explainable``
```

Comment:
Noted that we don't have a stub for the Explainable interface in the docs (nor do we need one), so code formatting is the right call here.